source/multigrid/mg_transfer_prebuilt.cc

```diff
@@ -236,6 +236,33 @@ void MGTransferPrebuilt<VectorType>::build_matrices
               }
           }
 
+      // Distribute sparsity pattern over all processes
+      // Retrieve communicator from triangulation if it is parallel
+      const parallel::Triangulation<DoFHandlerType::dimension,DoFHandlerType::space_dimension> *dist_tria =
```

Comment:
Would you mind to wrap this into a check whether the VectorType is `dealii::PETScWrappers::MPI::Vector`?

Reply:
`DoFHandlerType` is here simply `DoFHandler<dim,spacedim>`.

Reply:
What would be the best way to check the vector type? Personally I would prefer to do it in such a way that if people add other vector types later on requiring this, it doesn't grow too much. Perhaps a static bool variable in MatrixSelector?